source/getting-started/js-primer.md

```diff
@@ -55,31 +55,67 @@ export default Ember.Component.extend({
 
 ## Object Literal Shorthand
 
-In ES6 some functionality was added that allows the use of terser syntax for object literal properties and methods, along with some enhanced functionality for property names.
+ES6 introduced a new more succinct syntax for object literal properties and methods, along with some enhanced functionality for property names.
+
+### Property Initializer Syntax
 
-### Property intializer syntax
+Properties can now be set on object literals by including the name of the property only without the typical colon and value. This prevents some duplication when the object property is the same as the local variable you intend to set as the value of that property.
```

Comment:
I think we can expand this a little further. ES6's initializer syntax works if and only if the property name is the same as the variable holding the value to be assigned to it. For example, while this works
```javascript
let firstName = 'Ember';
let lastName = 'Tomster';

let tomster = {
  firstName,
  lastName
}
```
this won't
```javascript
let nombre = 'Ember';
let apellido = 'Tomster';

let tomster = {
  firstName,
  lastName
}
```
because the variables holding the values are different than the object's properties.